src/KintoID.sol

```diff
@@ -180,6 +180,25 @@ contract KintoID is
 
     /* ============ Burn ============ */
 
+    /**
+     * @dev Transfers the NFT from the old account to the new account
+     * @param _from Old address
+     * @param _to New address
+     */
+    function transferOnRecovery(address _from, address _to) external override {
+        require(balanceOf(_from) > 0 && balanceOf(_to) == 0, "Invalid transfer");
```

Comment:
```suggestion
        require(balanceOf(_from) > 0 && isSanctionsSafe(_from) balanceOf(_to) == 0, "Invalid transfer");
```

Reply:
not sure if we need this. You should be able to move it if you want to, you still cant transact

Reply:
well, by doing the other checks `isKYC()`, we are checking this. but yeah, not sure either, can you transfer a KintoID to an address that it's sanctioned?

Reply:
added it to be safe